src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -252,23 +253,32 @@ let printMessages (msgs: AnalyzerMessage list) =
 
     let msgLogger = factory.CreateLogger("")
 
-    msgs
-    |> Seq.iter (fun analyzerMessage ->
-        let m = analyzerMessage.Message
+    let useAnsiColorCodes = true
```

Comment:
Same as above, just always do the colored output for now.

Reply:
fixed now, thanks for the review!